api/vpc/v1alpha2/vpc_types.go

```diff
@@ -220,6 +228,7 @@ func (vpc *VPC) Default() {
 		if subnet.DHCP.Range.End == "" {
 			subnet.DHCP.Range.End = end
 		}
+		subnet.DHCP.InterfaceMTU = 9036 // Magic number should be named constant somewhere.
```

Comment:
in that case it'll always set value to 9036 ignoring the user input, should be done under the condition of it == 0